src/ngraph/frontend/onnx_import/onnx.hpp

```diff
@@ -61,17 +56,6 @@ namespace ngraph
                                    std::int64_t version,
                                    const std::string& domain = "ai.onnx");
 
-        /// \brief      Imports an in-memory ONNX model and converts it to an nGraph Function.
-        ///
-        /// \note       If the ONNX model contains unsupported ops, the function throws
-        ///             an ngraph_error exception.
-        ///
-        /// \param[in]  model_proto    The in-memory ONNX model reference
-        ///
-        /// \return     An nGraph function that represents a single output from the created graph.
-        NGRAPH_API
-        std::shared_ptr<Function> import_onnx_model(const onnx::ModelProto& model_proto);
```

Comment:
Why did you remove this method? Don't we need it anymore?

Reply:
Yes, I've removed it because we went back to the `std::stringstream` approach. Now with a custom namespace for ONNX in nGraph, the import problems are finally fixed.